src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/Window.cs

```diff
@@ -206,18 +206,18 @@ public bool TaskListDeleted
         }
 
         /// <summary>
-        /// Gets a value indicating whether determines whether the specified windows is the owner
+        /// Gets a value indicating whether the specified windows is the owner (i.e. doesn't have an owner)
         /// </summary>
         public bool IsOwner
         {
             get
             {
-                return NativeMethods.GetWindow(Hwnd, NativeMethods.GetWindowCmd.GW_OWNER) != null;
+                return NativeMethods.GetWindow(Hwnd, NativeMethods.GetWindowCmd.GW_OWNER) == IntPtr.Zero;
```

Comment:
Is this build breaking or a suggestion. If it's not build breaking it makes more sense if I fix it in my PR. I reworked the class and we will get conflicts.

Reply:
build breaking